kinto/config/kinto.tpl

```diff
@@ -11,49 +11,131 @@ port = %(http_port)s
 [app:main]
 use = egg:kinto
 
+# Production settings
+#
+# https://kinto.readthedocs.io/en/latest/configuration/production.html
+#
+# kinto.statsd_backend = kinto.core.statsd
+# kinto.statsd_url = udp://localhost:8125
+# kinto.statsd_prefix = kinto-prod
+
+# kinto.http_scheme = https
+# kinto.http_host = kinto.services.mozilla.com
+
+# kinto.backoff = 10
+# kinto.retry_after_seconds = 30
+# kinto.eos =
```

Comment:
These settings are already at the end. And I don't think most users will change them, so leave them at the end :)